src/UserMenu/UserMenu.js

```diff
@@ -125,7 +125,8 @@ const UserAvatar = ({ username, ...rest }) => {
       {...rest}
       sx={{
         background: "none",
-        border: theme => `2px solid ${theme.palette.background.paper}`,
+        border: "2px solid white",
```

Comment:
I guess this fixed the user avatar in both dark and light mode? Could you attach screenshot of each version just to confirm and to save me from having to manually build?

Reply:
Yes, changing the border-color resolved the issue.
dark mode:
![image](https://user-images.githubusercontent.com/49191249/173373827-f3eb2977-e311-4043-940d-a6a0c54a0b8a.png)
lightmode:
![image](https://user-images.githubusercontent.com/49191249/173374106-be956658-5602-4ef7-b05c-e966030a0f28.png)